visualization/app/codeCharta/util/codeMapHelper.ts

```diff
@@ -31,13 +31,29 @@ export function getNodesByGitignorePath(root: CodeMapNode, gitignorePath: string
 	if (gitignorePath.length === 0) {
 		return []
 	}
+	const ignoreResults = returnIgnore(gitignorePath)
+	const filtered = []
+	for (const { data } of hierarchy(root)) {
+		if (ignoreResults.ignoredNodePaths.ignores(transformPath(data.path)) === ignoreResults.condition) {
+			filtered.push(data)
+		}
+	}
+	return filtered
+}
+
+export function IsNodeExcludedOrFlattened(node: CodeMapNode, gitignorePath: string): boolean {
+	const ignoreResults = returnIgnore(gitignorePath)
+	return ignoreResults.ignoredNodePaths.ignores(transformPath(node.path)) === ignoreResults.condition
+}
+
+export function returnIgnore(gitignorePath: string) {
```

Comment:
Do we need to export this function? Seems like its never used in other parts, or am I wrong here?

Reply:
That is correct it is only used in this file.. I will remove the export keyword where its not needed.